xbmc/video/tags/VideoTagLoaderFFmpeg.cpp

```diff
@@ -39,8 +40,12 @@ CVideoTagLoaderFFmpeg::CVideoTagLoaderFFmpeg(const CFileItem& item,
   : IVideoInfoTagLoader(item, info, lookInFolder)
   , m_info(info)
 {
+  std::string filename =
+      item.IsStack() ? CStackDirectory::GetFirstStackedFile(item.GetPath()) : item.GetPath();
```

Comment:
A little late, but slightly more readable might be

```c++
std::string filename = item.GetPath();
if (item.IsStack())
  filename = CStackDirectory::GetFirstStackedFile(filename);
```

Reply:
I had an if in my first draft, but I preferred the direct assignment as it's pretty obvious imho.